test/ios_framework_import_test.sh

```diff
@@ -88,6 +88,50 @@ EOF
       "Payload/app.app/Frameworks/iOSDynamicFramework.framework/iOSDynamicFramework"
 }
 
+function create_swift_static_framework() {
+  if [[ -f frameworks/BUILD ]]; then
+    return
+  fi
+
+  mkdir libraries
+  mkdir frameworks
+
+  cat >> libraries/BUILD <<EOF
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
+swift_library(
+    name = "iOSSwiftStaticFrameworkLibrary",
+    module_name = "iOSSwiftStaticFramework",
+    srcs = ["@build_bazel_rules_apple//test/testdata/frameworks:swift_sources"],
+)
+EOF
+
+  do_build ios \
+    --compilation_mode=dbg \
+    --ios_minimum_os=11.0 \
```

Comment:
It would be nice to have this defined in a single place, any suggestions?

Reply:
Which part is "this" referring to, the minimum OS? Why do you want to move it elsewhere, I don't see it being used anywhere else?

Reply:
Yes the minimum OS.

The min OS has to be compatible with what the app uses, so this min OS is duplication.

Without specifying 11.0 as the min OS, the swiftmodule was defaulting to 13.1, which fails when being consumed by this test's app target, which is set to iOS 11.

Reply:
Right, the other usage was hidden by the collapsed part of the diff. (I'll never get used to GitHub's review interface.)

I'd just stash it in a variable at the top of this test file and use it throughout:

```
readonly IOS_VERSION=11.0

...

    minimum_os_version = "$IOS_VERSION",

...

    --ios_minimum_os="$IOS_VERSION" \
```

Reply:
That's what I was looking for, thanks.